src/s3ql/backends/swiftks.py

```diff
@@ -79,22 +79,22 @@ def _get_conn(self):
             tenant = None
             user = self.login
 
-        # We can optionally configure with tenant-name. If specified, it overrides tenant id from the login
-        tenant_name = self.options.get('tenant-name', None)
+        # We can optionally configure with tenant name instead of id.
+        tenant_is_name = self.options.get('tenant-is-name', False)
```

Comment:
Better use 
```suggestion
        tenant_is_name = 'tenant-is-name' in self.options
```
then it is consistent with other boolean backend options like `no-ssl` or `no-feature-detection`. (And the user does not need to provide a value to this option)

Reply:
Once again, great idea. Thanks for the good feedback. Done.